pkg/persistence/object_store_test.go

```diff
@@ -678,6 +696,73 @@ func TestNewObjectBackupStoreGetter(t *testing.T) {
 	}
 }
 
+// TestNewObjectBackupStoreGetterConfig runs the NewObjectBackupStoreGetter constructor and ensures
+// that it initializes the ObjectBackupStore with the correct config.
+func TestNewObjectBackupStoreGetterConfig(t *testing.T) {
```

Comment:
Would it be good to put these test cases under `TestNewObjectBackupStoreGetter`?

Reply:
I thought about that but the logic and set up for the tests is quite different so I opted to have a separate test for this.